cypress/test/tools/Tags.cy.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { mount } from 'cypress/react';
+import SelectTags from '../../../components/tools/Tags';
+
+describe('SelectTags', () => {
+  it('renders the component with the provided props', () => {
+    const name = 'Tag 1';
+    const bgColor = 'white';
+    const borderColor = 'border-blue-500';
+
+    mount(
+      <SelectTags name={ name } bgColor={ bgColor } borderColor={ borderColor } />
```

Comment:
Why component is named as `SelectTags` although we are using this component as `Tags` only?

Reply:
The component is exported as SelectTags